### ISSUE TITLE
Remove DASH

### ISSUE THREAD

**Issue by f8192** [tdickman/crypto51#45]
DASH had a network update which is focused to deal with 51% attack. From now on in order to perform a 51% attack you not only need to have 51% POW hashing power, but also to stake at least ~7200000 DASH within ~60% (7200) active masternodes. This is now impossible, since free 7200000 DASH don't exist yet. Thus, the cost of 51% attack cannot be calculated.

**Comment by tungfa** [comment]
agree to the above 
here are more details :
https://blog.dash.org/chainlocks-and-llmq-based-instantsend-mean-digital-cash-efe2852da9ca

**Comment by skaht** [comment]
@f8192 - There are currently 5010 masternodes (source Dash-Core wallet) today of which ~4900 masternodes are functioning properly (http://178.254.23.111/~pub/Dash/Dash_Info.html) to participate in long lived masternode quorums (LLMQ) per http://178.254.23.111/~pub/Dash/Dash_Info.html. The absolute "minimum cost" to control enough masternodes in an essentially randomly selected 24 hour BLS aggregate signed quorum today is (4900 masternodes) *(1000 Dash/masternodes)*($105/Dash) = $534.1M >> $0.890M that required to perform a 51% attack on BTC.

**Comment by f8192** [comment]
@skaht No, the minimum amount of masternodes to control for the attack is not 0.6*4900, it's (0.6/0.4)*4900. Existing masternodes are not going anywhere, you can't control them, as you need to deanonymize their owners first, and then persuade them to act maliciously or sell the masternodes to you. Impossible, even for governments!

**Comment by skaht** [comment]
@f8192 - See https://github.com/tdickman/crypto51/pull/44 for how the calculation is performed. However, if there is no collusion between existing masternode owners, the 51% attack price will go up significantly because of Dash market price increases due to substantially increased demand for Dash to acquire needed masternodes. Hence, the "minimum cost" to perform an attack.

**Comment by psypher246** [comment]
As mentioned in another thread. At the very least add Chainlocks to your list of possible solutions and reference the innovation to Dash

**Comment by greatwolf** [comment]
Any updates on this? The current masternode count that's active is ~4k. To get half that, let's say 2k masternodes that's still 2,000,000 dash. crypto51 needs to take that into account when calculating the cost to 51%.
